src/main/Catalog.js

```diff
@@ -1,27 +1,29 @@
 import React, { useState } from "react";
 
+const url = "https://api.punkapi.com/v2/beers";
+
 function Catalog() {
   let data;
-  const [state, setstate] = useState([]);
+  const [beerList, setBeerList] = useState([]);
```

Comment:
It's always good to have some sort of code ordering inside the component. Let's consider moving this `useState` to the 1st line and have everything else go below, with separation by empty lines.